xenon/cogs/basics.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, bot):
         await ctx.send(**ctx.em("**Invite Xenon**\n\n"
                                 f"[Xenon]({ctx.bot.invite})\n"
                                 "[Xenon Pro](https://discordapp.com/api/oauth2/authorize?client_id=524652984425250847&permissions=8&scope=bot) Use `x!pro` to get more information.\n"
-                                "[Xenon Turbo](https://discordapp.com/api/oauth2/authorize?client_id=598534174894194719&permissions=8&scope=bot)",
+                                "[Xenon Turbo](https://discordapp.com/api/oauth2/authorize?client_id=598534174894194719&permissions=8&scope=bot) Use `x!turbo` to get more information.",
```

Comment:
I think it would be batter removing both "use xxx to get more information." and adding "Use x!tiers to get more information" to the end.

Reply:
@ThatLifeofGames Do you want to update this?

Reply:
Doing it just as you said that